docs/dev-guide/modules.md

```diff
@@ -0,0 +1,32 @@
+# Modules
+
+Overview over all modules and dependencies.
+
+```mermaid
+flowchart BT
+
+ActivityModule --> ConfigModule
+ActivityModule --> ActivityDatabaseModule
+ActivityDatabaseModule --> ConfigModule
+BotModule --> ActivityDatabaseModule
+BotModule --> ConfigModule
+BotModule --> DiscordModule
+BotModule --> CabinModule
+BotModule --> EmbedModule
+CabinModule --> ConfigModule
+CabinModule --> CabinDatabaseModule
+CabinModule --> VisbookModule
+CabinDatabaseModule --> ConfigModule
+CabinDatabaseModule --> CabinUtModule
+CabinUtModule --> ConfigModule
+CabinUtModule --> VisbookModule
+DiscordModule --> ConfigModule
+DiscordModule --> CabinUtModule
+DiscordModule --> SlashCommandModule
+EmbedModule
+SlashCommandModule --> ConfigModule
+SlashCommandModule --> CabinModule
+SlashCommandModule --> DiscordModule
+SlashCommandModule --> EmbedModule
+VisbookModule --> ConfigModule
```

Comment:
Would it be worth dropping the config module as it just adds noise?

Reply:
I think that's a good idea, will remove it from the chart, it will make things easier to see.